src/client/layouts/default.vue

```diff
@@ -11,6 +11,8 @@
 </template>
 
 <script>
+import babelPolyfill from 'babel-polyfill'
```

Comment:
I think `import 'babel-polyfill'` would be clearer here, since we do not use the variable anywhere